test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -211,6 +222,19 @@ def test_successful_purchase_with_google_pay
     assert_match('google_pay', purchase.params.dig('charges', 'data')[0]['payment_method_details']['card']['wallet']['type'])
   end
 
+  def test_successful_purchase_with_tokenized_visa
+    options = {
+      currency: 'USD',
+      last_4: '4242'
+    }
+
+    purchase = @gateway.purchase(@amount, @network_token_credit_card, options)
+    assert_equal(nil, purchase.responses.first.params.dig('token', 'card', 'tokenization_method'))
+    assert purchase.success?
+    assert_not_nil(purchase.params.dig('charges', 'data')[0]['payment_method_details']['card']['network_token'])
+    # assert_true(purchase.params.dig('charges', 'data')[0]['payment_method_details']['card']['network_token']['used'])
```

Comment:
Waiting to hear back from Stripe about the `['payment_method_details']['card']['network_token']['used']` field.

Reply:
Update: I removed this assertion in [fda4b07](https://github.com/activemerchant/active_merchant/pull/4867/commits/fda4b075f8c5147035e34e49dd34b845e3b4212a) since Stripe currently only returns `true` for this field if network tokens are provisioned at Stripe.